src/main/java/uk/gov/register/presentation/HtmlViewSupport.java

```diff
@@ -28,6 +28,10 @@ public static String fieldLink(String fieldName) {
         return UriBuilder.fromPath("http://field.openregister.org/field").path(fieldName).build().toString();
     }
 
+    public static String publicBodyLink(String publicBodyId){
+        return UriBuilder.fromPath("http://public-body.openregister.org/public-body").path(publicBodyId).build().toString();
```

Comment:
Are you talking about LinkValue? We can not use LinkValue for this purpose as that is a FieldValue. we can use common template for register urls in the application so that changes are at one place but somewhere we have to hardcode public-body register.